Content.Server/Spreader/SpreaderSystem.cs

```diff
@@ -37,8 +38,7 @@ public sealed class SpreaderSystem : EntitySystem
 
     public const float SpreadCooldownSeconds = 1;
 
-    [ValidatePrototypeId<TagPrototype>]
-    private const string IgnoredTag = "SpreaderIgnore";
+    private static readonly ProtoId<TagPrototype> IgnoredTag = "SpreaderIgnore";
```

Comment:
for these will ProtoId have a compile time / initialization stage error if you do this or will it just silently have an invalid id

Reply:
yes, electro made a linter pass for static protoid variables

Reply:
```cs
/// <remarks>
///     This will be automatically validated by <see cref="ProtoIdSerializer{T}"/> if used in data fields.
/// </remarks>
```

Reply:
```cs
/// <summary>
/// This attribute should be used on static string or string collection fields to validate that they correspond to
/// valid YAML prototype ids. This attribute is not required for static <see cref="ProtoId{T}"/> and
/// <see cref="EntProtoId"/> fields, as they automatically get validated.
/// </summary>
[AttributeUsage(AttributeTargets.Field)]
public sealed class ValidatePrototypeIdAttribute<T> : Attribute where T : IPrototype
{
}
```